apps/cookbook/src/app/showcase/checkbox-showcase/checkbox-showcase.component.html

```diff
@@ -41,8 +41,16 @@ <h2>States</h2>
 
 <h2>Sizes</h2>
 <p>
-  The size applies to the clickable/tapable area (highlighted in the examples below). The checkbox
-  and checkmark always have the same size.
+  The small and medium sizes apply to the clickable/tapable area (highlighted in the examples below)
+  while the checkbox itself remains the same size.
+</p>
+<p>
+  The extra small size applies to both click-area and checkbox, but should be used with caution.
+</p>
+<p>
+  The extra small size is mainly intended for use in expert systems or information dense interfaces
+  where the reduced size is a necessity but note that it comes at the cost of degraded
+  accessibility.
 </p>
 <cookbook-example-viewer [html]="checkboxSizesExample.template">
   <cookbook-checkbox-sizes-example #checkboxSizesExample></cookbook-checkbox-sizes-example>
```

Comment:
Consider adding a divider inside the `cookbook-checkbox-sizes-example` so it looks something like this:

```
<kirby-checkbox size="xs" text="Extra Small"></kirby-checkbox>
<kirby-divider [hasMargin]="true"></kirby-divider>
<kirby-checkbox size="sm" text="Small"></kirby-checkbox>
<kirby-checkbox size="md" text="Medium (default)"></kirby-checkbox>
```

<img width="645" alt="image" src="https://github.com/kirbydesign/designsystem/assets/42470636/4d1d6aa4-a52e-48cc-a175-c037c1d9781d">